ndmg/stats/qa_fast.py

```diff
@@ -0,0 +1,207 @@
+    for i, coord in enumerate(coords):
+        for pos in coord:
+            idx += 1
+            ax = foverlay.add_subplot(3, 3, idx)
+            ax.set_title(var[i] + " = " + str(pos))
+            if i == 0:
+                image = ndimage.rotate(csf[pos, :, :], 90)
+                atl = ndimage.rotate(gm[pos, :, :], 90)
+                inim = ndimage.rotate(wm[pos, :, :], 90)
+            elif i == 1:
+                image = ndimage.rotate(csf[:, pos, :], 90)
+                atl = ndimage.rotate(gm[:, pos, :], 90)
+                inim = ndimage.rotate(wm[:, pos, :], 90)
+            else:
+                image = csf[:, :, pos]
+                atl = gm[:, :, pos]
+                inim = wm[:, :, pos]
+
+            if idx % 3 == 1:
+                ax.set_ylabel(labs[i])
+                ax.yaxis.set_ticks([0, image.shape[0] / 2, image.shape[0] - 1])
+                ax.xaxis.set_ticks([0, image.shape[1] / 2, image.shape[1] - 1])
+            
+            image = (image*255).astype(np.uint8)
+            atl = (atl*255).astype(np.uint8)
+            inim = (inim*255).astype(np.uint8)
+            image = pad_im(image, max(image.shape), 0)
+            atl = pad_im(atl, max(image.shape), 0)
+            inim = pad_im(inim, max(image.shape), 0)
+            
+            if edge:
+                image = edge_map(image).data
+                image[image > 0] = max_val
+                image[image == 0] = min_val
+            
+            plt.xticks([])
+            plt.yticks([])
+            ax.spines['top'].set_visible(False)
+            ax.spines['right'].set_visible(False)
+            ax.spines['bottom'].set_visible(False)
+            ax.spines['left'].set_visible(False)
+            ax.imshow(atl, interpolation="none", cmap=cmaps[0], alpha=0.65)
+            ax.imshow(inim, interpolation="none", cmap=cmaps[1], alpha=0.65)
+            ax.imshow(image, interpolation="none", cmap=cmaps[2], alpha=0.6)
+            ax.imshow(
+                opaque_colorscale(
+                    cmaps[0], atl, alpha=0.65, vmin=min_val, vmax=max_val
+                )
+            )
+            ax.imshow(
+                opaque_colorscale(
+                    cmaps[1], inim, alpha=0.65, vmin=min_val, vmax=max_val
+                )
+            )
+            ax.imshow(
+                opaque_colorscale(
+                    cmaps[2], image, alpha=0.6, vmin=min_val, vmax=max_val
+                )
+            )
+            if idx == 3:
+                plt.plot(0, 0, "-", c='green', label='wm')
+                plt.plot(0, 0, "-", c='pink', label='gm')
+                plt.plot(0, 0, "-", c='blue', label='csf')
+                plt.legend(loc='upper right',fontsize=15,bbox_to_anchor=(1.5,1.5))
+            
+    foverlay.set_size_inches(12.5, 10.5, forward=True)
+    foverlay.tight_layout
+    return foverlay
+
+def get_min_max(data, minthr=2, maxthr=95):
+    """
+    data: regmri data to threshold.
+    """
```

Comment:
better docstrings, as discussed in other places

Reply:
if this function is being used by the other qa, it's another one that should be consolidated